### ISSUE TITLE
Unexpected Error post 1.4.281.0 patch

### ISSUE THREAD

**Issue by GidonsClaw** [lisongmechlab/lsml#822]
> Before reporting please:
> 1. Check that you have the latest version of LSML installed by visiting http://li-soft.org.
> 2. Check that the `core_gameInstallDir` entry in `%APPDATA%\LiSoft\LSML` points to the correct game install (if you have one).
> 3. Use the search field at the top of this page and search to see if the issue is already reported.
>
> To make fixing the bug as fast as possible, please fill in the following information as precisely as possible.
> This is necessary because people think in different ways, what is obvious to you may not be obvious to me. Please don't leave any steps out because the exact steps give me insight into the problem that help me fix it faster.
> 
> PLEASE REMOVE THIS LINE AND ANYTHING ABOVE IT!

**LSML version:** x.y.z
**Screen resolution:** 1920x1200
**Operating System:** Windows 10

**To Reproduce**
1. Open LSML

**Actual result:**
On opening LSML, the following errors:

**Error 1**
```
No chassis variation named: 3607 !
Cause:
`com.thoughtworks.xstream.converters.ConversionException: 
---- Debugging information ----
cause-exception     : java.lang.IllegalArgumentException
cause-message       : No chassis variation named: 3607 !
class               : org.lisoft.lsml.model.loadout.LoadoutStandard
required-type       : org.lisoft.lsml.model.loadout.LoadoutStandard
converter-type      : org.lisoft.lsml.model.export.garage.LoadoutConverter
line number         : 71185
class[1]            : java.util.ArrayList
required-type[1]    : java.util.ArrayList
converter-type[1]   : com.thoughtworks.xstream.converters.collections.CollectionConverter
class[2]            : org.lisoft.lsml.model.garage.GarageDirectory
required-type[2]    : org.lisoft.lsml.model.garage.GarageDirectory
converter-type[2]   : com.thoughtworks.xstream.converters.reflection.ReflectionConverter
class[3]            : org.lisoft.lsml.model.garage.Garage
required-type[3]    : org.lisoft.lsml.model.garage.Garage
converter-type[3]   : org.lisoft.lsml.model.export.garage.GarageConverter
version             : not available
-------------------------------
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readBareItem(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readItem(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.addCurrentElementToCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshallField(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.doUnmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readBareItem(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readItem(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.addCurrentElementToCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshallField(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.doUnmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readBareItem(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readItem(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.addCurrentElementToCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshallField(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.doUnmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readBareItem(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.AbstractCollectionConverter.readItem(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.addCurrentElementToCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.populateCollection(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.collections.CollectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshallField(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.doUnmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshallField(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.doUnmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshal(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.model.export.garage.GarageConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.start(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.AbstractTreeMarshallingStrategy.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.XStream.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.XStream.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.XStream.fromXML(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.model.garage.GarageSerializer.load(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.GlobalGarage$GarageOnDisk.<init>(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.GlobalGarage.openLastGarage(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.LiSongMechLab.foregroundLoad(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.LiSongMechLab.lambda$start$1(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.DialogErrorReporter.informUser(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.DialogErrorReporter.error(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.DialogErrorReporter.lambda$error$1(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: No chassis variation named: 3607 !
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.model.ChassisDB.lookup(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.model.export.garage.LoadoutConverter.parseV2(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.model.export.garage.LoadoutConverter.unmarshal(Unknown Source)
	... 109 more`

**Error 2**
Unable to parse game  files!
Cause:
`org.lisoft.mwo_data.mwo_parser.ParseErrorException: Unable to parse game files!
	at org.lisoft.lsml@2.1.0/org.lisoft.mwo_data.mwo_parser.MwoDataReader.parseGameFiles(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.GraphicalDatabaseProvider.updateDatabase(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.GraphicalDatabaseProvider.loadDatabase(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.GraphicalDatabaseProvider.getDatabase(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.LiSongMechLab.backgroundLoad(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.LiSongMechLab$1.call(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.lsml.view_fx.LiSongMechLab$1.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: Unable to load chassis configuration! Chassis: viper
	at org.lisoft.lsml@2.1.0/org.lisoft.mwo_data.mwo_parser.PartialDatabase.parseOmniPods(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.mwo_data.mwo_parser.PartialDatabase.generateDatabase(Unknown Source)
	... 10 more
Caused by: com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter$DuplicateFieldException: Duplicate field Bonus
---- Debugging information ----
message             : Duplicate field Bonus
field               : Bonus
class               : org.lisoft.mwo_data.mwo_parser.XMLOmniPods$XMLOmniPodsSet$XMLOmniPodsSetBonuses
required-type       : org.lisoft.mwo_data.mwo_parser.XMLOmniPods$XMLOmniPodsSet$XMLOmniPodsSetBonuses
converter-type      : com.thoughtworks.xstream.converters.reflection.ReflectionConverter
path                : /OmniPods/Set[8]/SetBonuses/Bonus[2]
line number         : 364
class[1]            : org.lisoft.mwo_data.mwo_parser.XMLOmniPods$XMLOmniPodsSet
required-type[1]    : org.lisoft.mwo_data.mwo_parser.XMLOmniPods$XMLOmniPodsSet
class[2]            : org.lisoft.mwo_data.mwo_parser.XMLOmniPods
required-type[2]    : org.lisoft.mwo_data.mwo_parser.XMLOmniPods
version             : not available
-------------------------------
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter$3.add(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.doUnmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshallField(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.doUnmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.doUnmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.converters.reflection.AbstractReflectionConverter.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.AbstractReferenceUnmarshaller.convert(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.convertAnother(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.TreeUnmarshaller.start(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.core.AbstractTreeMarshallingStrategy.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.XStream.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.XStream.unmarshal(Unknown Source)
	at lsml.merged.module@2.1.0/com.thoughtworks.xstream.XStream.fromXML(Unknown Source)
	at org.lisoft.lsml@2.1.0/org.lisoft.mwo_data.mwo_parser.XMLOmniPods.fromXml(Unknown Source)
	... 12 more`
```
**Expected behavior**
Normal Loading

**Additional context**
PGI Added more mechs, but especially, new weapons like the HAG, Binary Laser and X-Pulse lasers.

**Comment by sestrand** [comment]
also, related to the new weapons.

Unknown spec: chag40 in quirk: ammocapacity_chag40_additive

**Comment by LiSongMWO** [comment]
I'll take a look soon, I've been out of country on a long trip...

**Comment by LiSongMWO** [comment]
Fixed for 2.1.1
